packages/legend-graph/src/graph-manager/protocol/pure/v1/engine/analytics/V1_MappingModelCoverageAnalysis.ts

```diff
@@ -144,6 +144,13 @@ export class V1_MappingModelCoverageAnalysisResult {
   );
 }
 
+export const V1_mappingModelCoverageAnalysisResultModelSchema =
```

Comment:
I thought we already have `V1_MappingModelCoverageAnalysisResult.serialization`?